RocketControlUnitGUI/src/data.ts

```diff
@@ -0,0 +1,110 @@
+  // data.ts
+  import PocketBase from 'pocketbase';
+  import dotenv from "dotenv";
+
+  
+
+  const ADMIN_EMAIL: string = import.meta.env.VITE_EMAIL;
+  const ADMIN_PASSWORD: string = import.meta.env.VITE_PASSWORD;
+
+ console.log(ADMIN_EMAIL);
+
+  const pb = new PocketBase('http://127.0.0.1:8090');
+  pb.autoCancellation(false);
+
+  let isAuthenticated = false;
+
+
+
+  // Function to authenticate the admin user
+  export async function authenticate() {
```

Comment:
Authentication as well as subscribing is handled in usePocketbase hook (which may need some additions) in master so refactor to use that. Also, when using the hook we should make sure to pass around a global instance so that we are not opening duplicate pb connections like in this file.

Reply:
I have to refactor my code to incorporate the usePocketbase hook now will I will be implementing soon!